### DISCUSSION TITLE
ATDD :개발자, QE, PO 모두 속이 편해지는  인수 테스트 주도 개발 적용 해보기

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#126] [General]
태형님 발표자료 수정했다고 합니다.
의견 있으면 댓글로 다시면 좋을 것 같아요.

https://docs.google.com/presentation/d/18M4__L6e8hL8ahEdkkMvVdC4BSx0LP-NITnmjBGK5Dg/edit?usp=sharing

**Comment by hi-hyein** [comment]
@TaeHyoungKwon 
바쁘실 텐데 편집까지 하셔서 귀한 자료 공유해 주신 태형님께 다시 한번 감사의 말씀을 드립니다.
감사합니다!

**Comment by wooyaggo86** [comment]
태형님 자료 공유해주셔서 감사해요!🙏🙏👍

**Comment by Joseph-Cha** [comment]
Discussion 내용을 이제서야 확인을 했습니다..ㅠㅠ
자료 공유 너무 감사합니다👍👍👍

**Comment by momoci99** [comment]
자료 공유해주셔서 감사합니다 ㅜㅜ
